docs/user/reference/environment-variables.rst

```diff
@@ -0,0 +1,50 @@
+Environment variables
+=====================
+
+All :doc:`build processes </builds>` have the following environment variables automatically defined and available for each build step:
+
+.. envvar:: READTHEDOCS
+
+    Whether the build is running inside Read the Docs.
+
+    :Default: ``True``
+
+.. envvar:: READTHEDOCS_LANGUAGE
+
+    The locale name, or the identifier for the locale, for the project being built.
+    This value comes from the project's configured language.
+
+    :Examples: ``en``, ``it``, ``de_AT``, ``es``, ``pt_BR``
+
+.. envvar:: READTHEDOCS_PROJECT
```

Comment:
I'd put this above language too, since it relates to the project. I always think about them as the order in the URL:

`<project>.readthedocs.io/<language>/<version>/`